dvc/fs/local.py

```diff
@@ -107,6 +107,9 @@ def rm_file(self, path):
     def rm(self, path, recursive=False, maxdepth=None):
         remove(path)
 
+    def cp_file(self, path1, path2, **kwargs):
```

Comment:
Right now, `cp` and `cp_file` in LocalFileSystem is same, i.e non-recursive. I'll fix that later. :)